wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -76,7 +76,7 @@ Overall rules and guidelines apply to every kind of osu! difficulty. Rhythm rela
 -   **Buzz sliders should have appropriate delay before the next note.** 1/8 and 1/16 sliders should be followed by a 1/4 gap, whereas 1/12 sliders should be followed by a 1/6 gap. This ensures that the hit-window between hit objects is playable.
 -   **Spinners should be used when they fit the music.** This is to ensure score differences among perfect plays on the leaderboard. They usually fit during held notes, changes in intensity, or transitions between sections.
 -   **Avoid using similar combo colours in successive combos.** Players should be able to differentiate between different combos in a map.
--   **Spinners and sliderends should have hit sound feedback.** If either are used to represent a held sound and do not end on a distinct sound, having no feedback is acceptable.
+-   **Spinner ends, slider ends, and slider reverses should have hit sound feedback.** If these are used to represent a held sound and do not align with a distinct sound, having no feedback is acceptable.
```

Comment:
`hit sound` -> `hitsound`

Missed this article.

Reply:
okoratu advised towards hit sound, so i'll go with that for now. if the decision is made to use hitsound, i'll change accordingly